package.json

```diff
@@ -53,7 +53,7 @@
     "tslint": "^4.0.0",
     "tslint-config-ktsn": "^2.0.0",
     "typescript": "^2.1.4",
-    "vue": "^2.1.10",
+    "vue": "^2.2.6",
```

Comment:
Then, I think we should support 2.2.0 or later because esm build is bundled since 2.2.0.
https://github.com/vuejs/vue/releases/tag/v2.2.0

> And Vue 2.2.0 does not accept this

I guess this is because v2.2.0 actually does not have esm build in dist directory, this was fixed on v2.2.1.